src/Refactoring-Core/RBExtractMethodAndOccurrences.class.st

```diff
@@ -69,7 +69,7 @@ RBExtractMethodAndOccurrences >> findOccurrencesOf: aSelector [
 			model: self model 
 			find: aSelector
 			of: class
-			inAllHierarchy: true.
+			inAllHierarchy: (self options at: #searchInAllHierarchy) value.
```

Comment:
I know that this is like this before, but maybe we want to use a nicer object for the options.
Something that wraps a dictionary can be an initial start, but for example that we can send the message #searchInAllHierarchy and not just like a symbol. Using symbols seems nice but it creates kind of strange coupling between both parts of the system. Also, I have a question, why the #value message?

Reply:
umm... ok, yeah we can improve that, for the moment NautilusRefactoring initialize many options, even overridden some options changed inside commands. I'll think how improve this. I call #value method to execute the block (of that option) and get a boolean from user.

Reply:
Yes this is a bit the mess.